Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics.MicroGraphics/Driver/GraphicsPath.cs

```diff
@@ -76,26 +76,27 @@ public Rect Bounds
         {
             get
             {
-                if (Points == null)
+                if (PathActions == null || PathActions.Count == 0)
```

Comment:
What happens if there are just 1 or 2 points?  It returns a rect with all values the same, or with no area (i.e. a line)?

Reply:
I think it still technically defines the bounds of he path correctly